x-pack/plugins/security_solution/kibana.json

```diff
@@ -36,9 +36,7 @@
     "usageCollection",
     "lens",
     "lists",
-    "home",
-    "telemetry",
-    "telemetryManagementSection"
```

Comment:
Did you want to remove `telemetry` plugin altogether here or just  `telemetryManagementSection`. We use `telemetry`